tests/conftest.py

```diff
@@ -170,6 +171,9 @@ def pytest_addoption(parser):
     # qos_sai options
     parser.addoption("--ptf_portmap", action="store", default=None, type=str, help="PTF port index to DUT port alias map")
 
+    # Kubernetes master options
+    parser.addoption("--kube_master", action="store", default=None, type=str, help="Kubernetes master identifier in the form {servernumber}_{mastersetnumber}")
```

Comment:
This help text needs to be updated as well.

Reply:
Updated